### PR TITLE
Create test2.bedgraph

### PR DESCRIPTION
**Pull request by ekushele**
Add another bedgraph file

### CODE REVIEW

**Comment by nvnieuwk** [comment]
Can you add this file to the README?

**Comment by ekushele** [comment]
> Can you add this file to the README?

Added..

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ If you cannot find suitable test data on this repository, please contact us on t
       - 'read_group_settings.txt': a simple textfile containing the readgroup settings '1 paired' primarily used for the atlas/splitmerge module
     - bedgraph
       - 'test.bedgraph'
+      - 'test2.bedgraph'
```
[the remark this answers is not in the document]
Comment (nvnieuwk) [reply README.md]: Allright, it's not advised to just duplicate data on here though. You can either change the name of the first file in your test script or you can add a different file here instead

[the remark this answers is not in the document]
Comment (nvnieuwk) [reply README.md]: Something like this should work:

```nextflow
channel_with_bedgraph.splitText().collectFile(name:"name_of_new_file.bedgraph")
```

**Pull request closed without merging**